app/views/user_mailer/invitation.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <p>
     <% if !@user_name.blank? %>
-      <%= t ".has_invited", user_name: style_bold(@user_name) %>
+      <%= t ".has_invited", user_name: style_bold(@user_name).html_safe %>
```

Comment:
This needs to be `<%= t(".has_invited", user_name: style_bold(@user_name)).html_safe %>` or it will still produce escaped HTML